impl/src/main/java/jakarta/faces/application/FacesMessage.java

```diff
@@ -135,13 +133,13 @@ public class FacesMessage implements Serializable {
      * order of their ordinal value.
      * </p>
      */
-    public static final List VALUES = unmodifiableList(asList(values));
+    public static final List<Severity> VALUES = List.of(values);
```

Comment:
This change from `List` to `List<Severity>` is an API change and must be done via a separate ticket/PR in https://github.com/jakartaee/faces/issues and take into account https://github.com/jakartaee/faces/issues?q=is%3Aissue+is%3Aopen+facesmessage+severity